### PR TITLE
Bl 144 fe remove question from mentee intake form

### PR DESCRIPTION
**Pull request by Nicholas-R-Payne**
## Description

Fixes # BL-144
[Link to Jira ticket](https://bloomtechlabs.atlassian.net/browse/BL-144)

- Verified question regarding experience level had been removed from mentee intake form.
- Removed "Experience Level:" from mentee review application.

## Loom Video

https://www.loom.com/share/c5871468e904467ba7480353813ca220

## Type of change

Please delete options that are not relevant.

- [ x ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x ] I have removed unnecessary comments/console logs from my code
- [ x ] I have made corresponding changes to the documentation if necessary (optional)
- [ x ] My changes generate no new warnings
- [ x ] I have checked my code and corrected any misspellings
- [ x ] No duplicate code left within changed files
- [ x ] Size of pull request kept to a minimum
- [ x ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by Charles-Merrill-Johnson — APPROVED** [review]
Looks good. A pretty straightforward process here. Good job on making sure the application review, reflected the same input fields as the form itself.

**Review by MelodyLayne — APPROVED** [review]

File: src/components/pages/PendingApplications/ApplicationModal.js [hunk 1]
```diff
@@ -170,9 +170,6 @@ const ApplicationModal = ({
                 <b>Location:</b> {currentApplication.city},{' '}
                 {currentApplication.state} {currentApplication.country}
               </List.Item>
-              <List.Item>
-                <b>Experience Level:</b> {currentApplication.experience_level}
-              </List.Item>
```
Comment (ElijahHopkin) [inline src/components/pages/PendingApplications/ApplicationModal.js:175]: Thanks for catching this bug that was clearly missed earlier! I'm also thrilled that it's such an easy fix!

**Review by ElijahHopkin — APPROVED** [review]
simple fix to a woopsie. Thanks for catching it AND for following through even though it appeared fixed, at first.

**Review by jettyblue — APPROVED** [review]
Nice catch and fix. Looks good, well done!

**Review by nathancai97 — DISMISSED** [review]
Good catch on removing this code! This could've been confusing as a mentor but now the modal is all lined up.

**Review by miguelaledesma — DISMISSED** [review]
Thanks for removing the fields that are no longer necessary. Great work leaving the code nice and clean. Great Job.

**Review by techlando — DISMISSED** [review]
nice cleanup... some checks are failing but good job

**Review by brimjustin — APPROVED** [review]
Nice job, the redundancy of the question allows for mentors to have a better experience.

File: .vs/ProjectSettings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```
Comment (Johnjx) [inline .vs/ProjectSettings.json:3]: Are we aware of why these `vs` files have been altered/added?

**Review by Johnjx — COMMENTED** [review]
Good catch, is PG Admin DB also in line with the removal goals here?

File: .vs/ProjectSettings.json (hunk 2 above)
Comment (Nicholas-R-Payne) [reply .vs/ProjectSettings.json]: Thank you for catching that, I don't know how those were added in.

**Review by brimjustin — APPROVED** [review]
Nice job, the redundancy of the question allows for mentors to have a better experience.

**Review by MelodyLayne — APPROVED** [review]

File: .vs/ProjectSettings.json (hunk 2 above)
Comment (Johnjx) [reply .vs/ProjectSettings.json]: No problem, there are all kinds of settings going on, hard to keep track of sometimes.

**Review by albert-szabo — APPROVED** [review]
A good, straightforward fix! This change should help avoid confusion and maintain consistency.

**Review by Charles-Merrill-Johnson — APPROVED** [review]
Attention to detail is an important quality and this is a great example of that by catching that bug! Great Job!

File: src/components/pages/PendingApplications/ApplicationModal.js (hunk 1 above)
Comment (ElijahHopkin) [reply src/components/pages/PendingApplications/ApplicationModal.js:175]: just following up on my comment because it's not being recorded as a review.

**Review by ElijahHopkin — APPROVED** [review]

**Pull request merged**